python/invoke_release/tasks.py

```diff
@@ -47,6 +49,8 @@
 
 __POST_APPLY = False
 
+GITHUB_PULL_URL = ("%s/repos/%s/pulls")
```

Comment:
You can just add `https://api.github.com` in here. I don't think that will change